package.json

```diff
@@ -40,7 +40,6 @@
     "@react-native-community/masked-view": "0.1.10",
     "@react-native-community/netinfo": "5.9.9",
     "@react-navigation/bottom-tabs": "5.11.8",
-    "@react-navigation/drawer": "5.12.5",
```

Comment:
why removing this?

Reply:
there are no imports for `@react-navigation/drawer`
why keep it around?